ElementX/Sources/Services/Client/ClientProxy.swift

```diff
@@ -141,6 +141,31 @@ class ClientProxy: ClientProxyProtocol {
         slidingSyncObserverToken = nil
     }
     
+    func directRoomForUserID(_ userID: String) async -> Result<String?, ClientProxyError> {
+        await Task.dispatch(on: clientQueue) {
+            do {
+                if let room = try self.client.getDmRoom(userId: userID) {
+                    return .success(room.id())
+                }
+            } catch {
+                return .failure(.failedRetrievingDirectRoom)
+            }
+            return .success(nil)
+        }
+    }
+    
+    func createDirectRoom(with userID: String) async -> Result<String, ClientProxyError> {
+        await Task.dispatch(on: clientQueue) {
+            do {
+                let parameters = CreateRoomParameters(name: "", topic: nil, isEncrypted: true, isDirect: true, visibility: .private, preset: .privateChat, invite: [userID], avatar: nil)
```

Comment:
```suggestion
                let parameters = CreateRoomParameters(name: "", topic: nil, isEncrypted: true, isDirect: true, visibility: .private, preset: .trustedPrivateChat, invite: [userID], avatar: nil)
```

I think trusted private chat is the default on Element legacy. Please have a look there. 
I think it makes more sense.